loader.js

```diff
@@ -0,0 +1,56 @@
+var carContainer;
+var counter = null
+var userInput = document.querySelector('input')
+var CarLot = (function () {
+  var inventory = [];
+  return {
+    getInventory: function () {
+      return inventory
+    },
+    loadInventory: function (callbackFunction) {
+      var inventoryLoader = new XMLHttpRequest();
+      inventoryLoader.open('GET', "inventory.json");
+      inventoryLoader.send();
+      inventoryLoader.addEventListener("load", function () {
+      inventory = JSON.parse(this.responseText).cars;
+      callbackFunction(inventory)
+      });
+    }
+  };
+  return CarLot
+})();
+
+CarLot.loadInventory(populateLot);
```

Comment:
Not sure why this and the `populateLot` function got moved from `quiz.js` and into this IIFE, as the boilerpplate code in the README sets up.